Wry 0.16.1 does not build on macos

**Describe the bug**
The build of the crate fails on macos.

This is the error message:
```
error[E0308]: mismatched types
   --> /Users/me/.cargo/registry/src/github.com-1ecc6299db9ec823/wry-0.16.1/src/webview/wkwebview/mod.rs:420:12
    |
420 |         if can_set_titlebar_style {
    |            ^^^^^^^^^^^^^^^^^^^^^^ expected `bool`, found `i8`

error[E0308]: mismatched types
   --> /Users/me/.cargo/registry/src/github.com-1ecc6299db9ec823/wry-0.16.1/src/webview/wkwebview/mod.rs:549:10
    |
549 |       if can_print {
    |          ^^^^^^^^^ expected `bool`, found `i8`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `wry` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Steps To Reproduce**
Try to build the crate using `cargo build`.

**Expected behavior**
The build should succeed.

**Platform and Versions (please complete the following information):**
OS: MacOS 12.3.1
Rustc: rustc 1.60.0 (7737e0b5c 2022-04-04)

**Would you assign yourself to resolve this bug?**
- [x] Yes
- [ ] No

**Additional context**
The bug seems to have been introduced in eb2dddb611f7fadf35bf7d7c32cb6d054da9fe9e.

However I have no idea how it was possible, that this slipped the Github Actions CI?

--- a/.changes/macos_build.md
+++ b/.changes/macos_build.md
@@ -1,0 +1,5 @@
+---
+"wry": patch
+---
+
+Fixed build on macos.

--- a/src/webview/wkwebview/mod.rs
+++ b/src/webview/wkwebview/mod.rs
@@ -417,7 +417,7 @@ impl InnerWebView {
           ns_window,
           respondsToSelector: sel!(setTitlebarSeparatorStyle:)
         ];
-        if can_set_titlebar_style {
+        if can_set_titlebar_style != NO {
           // `1` means `none`, see https://developer.apple.com/documentation/appkit/nstitlebarseparatorstyle/none
           let () = msg_send![ns_window, setTitlebarSeparatorStyle: 1];
         }
@@ -546,7 +546,7 @@ r#"Object.defineProperty(window, 'ipc', {
         self.webview,
         respondsToSelector: sel!(printOperationWithPrintInfo:)
       ];
-      if can_print {
+      if can_print != NO {
         // Create a shared print info
         let print_info: id = msg_send![class!(NSPrintInfo), sharedPrintInfo];
         let print_info: id = msg_send![print_info, init];

--- a/src/webview/wkwebview/mod.rs
+++ b/src/webview/wkwebview/mod.rs
@@ -546,7 +546,7 @@ r#"Object.defineProperty(window, 'ipc', {
         self.webview,
         respondsToSelector: sel!(printOperationWithPrintInfo:)
       ];
-      if can_print != NO {
+      if can_print == YES {
         // Create a shared print info
         let print_info: id = msg_send![class!(NSPrintInfo), sharedPrintInfo];
         let print_info: id = msg_send![print_info, init];

--- a/src/webview/wkwebview/mod.rs
+++ b/src/webview/wkwebview/mod.rs
@@ -417,7 +417,7 @@ impl InnerWebView {
           ns_window,
           respondsToSelector: sel!(setTitlebarSeparatorStyle:)
         ];
-        if can_set_titlebar_style != NO {
+        if can_set_titlebar_style == YES {
           // `1` means `none`, see https://developer.apple.com/documentation/appkit/nstitlebarseparatorstyle/none
           let () = msg_send![ns_window, setTitlebarSeparatorStyle: 1];
         }

